### DISCUSSION TITLE
Figma Integration (App)

### DISCUSSION THREAD

**Discussion by andrewbrown00** [mattermost/community#33] [Practical Ideas]
Figma is the tool we use to design, test, and iterate designs from end to end. Having a Figma app for Mattermost allows teams to stay in sync and up-to-date on the latest design work. It would provide a way to easily share Figma design files in MM and personalized notifications when:

- you're @mentioned in comments in a Figma file
- some replies to one of your comments
- you're invited to teams, projects, and files
- and more

**Comment by esethna** [comment]
YAS! I heard @michaelgamble was working on this ;)

**Reply by matthewbirtch** [reply]
100%. @michaelgamble and I started this doc in confluence: https://mattermost.atlassian.net/wiki/spaces/FPFM/pages/1180827667/Figma+Plugin+For+Mattermost
